manifests/init.pp

```diff
@@ -36,6 +36,38 @@
   }
   validate_string($tac_plus_template_real)
 
+  if $acl == 'NONE' {
```

Comment:
hahahaha, can hear the whines and whistles already ;)

I didn't add test to proof that functionality with the intention to remove it with one of the next versions again. So that there is a short grace period only.